test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
I'm not comfortable with that. At the end it's still `Sidekiq::Web` to mount.
Adding instance methods adds complexity for the sole purpose to be tested.

That's my opinion. If you prefer tests with instance methods, the commit is ready.